src/NLog/LayoutRenderers/ExceptionLayoutRenderer.cs

```diff
@@ -66,8 +73,6 @@ public ExceptionLayoutRenderer()
             this.MaxInnerExceptionLevel = 0;
         }
 
-        private delegate void ExceptionDataTarget(StringBuilder sb, Exception ex);
-
         /// <summary>
         /// Gets or sets the format of the output. Must be a comma-separated list of exception
```

Comment:
Is this one you are looking for?

~~~~~~~~~~~~~~~~~~~~~
/// <summary>
        /// Gets or sets the format of the output. Must be a comma-separated list of exception
        /// properties: Message, Type, ShortType, ToString, Method, StackTrace.
        /// This parameter value is case-insensitive.
        /// </summary>
        /// <see cref="Formats"/>
        /// <see cref="ExceptionRenderingFormat"/>
        /// <docgen category='Rendering Options' order='10' />
        [DefaultParameter]
        public string Format
        {
            get
            {
                return this.format;
            }

            set
            {
                this.format = value;
                Formats = CompileFormat(value);
            }
        }